templates/ado-resources/pipelines/build.yml

```diff
@@ -0,0 +1,80 @@
+trigger:
+  branches:
+    include:
+      - main
+
+pool:
+  vmImage: ubuntu-latest
+
+variables:
+  pnpm_config_cache: $(Pipeline.Workspace)/.pnpm-store
+
+stages:
+- stage: 'Build'
```

Comment:
Are "double quotes" supported in yaml files?

Of so, this is usually our standard.

Reply:
Yes, both are supported.

Globally, the choice I do here was to use single quote because initially, I have more single quote than double quote in the file.

I will update that 👍